src/utils/helpers.js

```diff
@@ -22,7 +22,8 @@ export const modalInstructions = {
   missionControl3: "It's time to join your squad! Click next to continue",
   sharePoints:
     "Ready Squad! Read your partner's story, view their drawing and share some points.",
-  matchUp: "Welcome to this week's matchup. Please vote 3 times.",
+  matchUp:
+    "Welcome to this week's matchup. Please vote 3 times to unlock matchup scores. You may continue voting up to 10 times.",
 };
```

Comment:
eventually we might want to either reword this so its shorter an/or disable the modal when returning to the dashboard, because reading this message 10 times could be cumbersome from a UX perspective. but definitely good idea to let the user know they have the ability to vote more than 3x. 

another thought: is there a way to make this dynamic so after voting once the message says "please vote 2 times" then "... 1 time" and after that saying "you have 7 votes remaining", "...6 votes..." etc? could shorten the message and provide real time info for user.